.github/workflows/pr_validation.yml

```diff
@@ -21,10 +21,12 @@ on:
     branches:
       - master
       - dev
+      - v1.4
```

Comment:
`v1.*` (for future-proofing)

Reply:
Good idea, didn't know that this is an option